src/com/xilinx/rapidwright/rwroute/GlobalSignalRouting.java

```diff
@@ -279,15 +279,21 @@ private static ClockRegion findCentroid(Net clk, Device device) {
         return device.getClockRegion(center.y, center.x);
     }
 
+    public enum NodeStatus {
```

Comment:
I know its a hassle, but consider possibly moving this to its own file.  Nested classes/enums create some untidiness in compilation.